_templates/navbar.html

```diff
@@ -1,7 +1,6 @@
-<a href="{{pathto('install')}}">Install</a> · 
-<a href="{{pathto('documentation')}}">Docs</a> ·
-<a href="{{pathto('videos')}}">Videos</a> · 
+<a href="http://jupyter.readthedocs.org/en/latest/install.html" target="_blank">Install</a> · 
```

Comment:
I thought we had decided that we would use the jupyter landing pages for both projects. The problem is that with ipython now also in mulitple repos, sending them to the ipython/ipython docs will be a deadend for users who want the ipywidgets, ipyparallel. I see two ways of solving this:

* Make a separately landing page for ipython docs that points to all of the different ipython repos/packages.
* Using the jupyter landing page for all.

One thing that we are seeing in the user tests is that it is very difficult for users to nav between the different readthedocs sites because there isn't a global nav element that spans everything. That is really is problem our docs landing page is trying to solve. I have some ideas on ways we can add that global nav element to all of our docs repos, but until then, I think linking to the global jupyter landing page makes sense.

Also, based on the google analytics of the old website, almost all users who arrive on the site are using the notebook so sending them to the ipython kernel docs alone, doesn't make much sense.